INSTALL

```diff
@@ -0,0 +1,9 @@
+# INSTALL / BUILD instruction for Apache Airflow (incubating)
+# fetch the tarball and untar the source
+
+# run Apache RAT (release audit tool)
+# RAT docs here: https://creadur.apache.org/rat/
+java -jar apache-rat.jar -E ./.rat-excludes -d .
```

Comment:
We expect users to install off of Pypi, this is for the source code release that the Apache community will vote on.